exercises/book-store/canonical-data.json

```
@@ -145,27 +147,33 @@
       "description": "Three each of first 2 books and 2 each of remaining books",
       "comments": ["Suggested grouping, [[1,2,3,4,5],[1,2,3,4,5],[1,2]]."],
       "input": {
-        "basket": [1,1,2,2,3,3,4,4,5,5,1,2]
+        "basket": [1, 1, 2, 2, 3, 3, 4, 4, 5, 5, 1, 2]
       },
       "expected": 7520
     },
     {
       "uuid": "bb376344-4fb2-49ab-ab85-e38d8354a58d",
       "property": "total",
       "description": "Four groups of four are cheaper than two groups each of five and three",
-      "comments": ["Suggested grouping, [[1,2,3,4],[1,2,3,5],[1,2,3,4],[1,2,3,5]]."],
+      "comments": [
+        "Suggested grouping, [[1,2,3,4],[1,2,3,5],[1,2,3,4],[1,2,3,5]]."
+      ],
       "input": {
-        "basket": [1,1,2,2,3,3,4,5,1,1,2,2,3,3,4,5]
+        "basket": [1, 1, 2, 2, 3, 3, 4, 5, 1, 1, 2, 2, 3, 3, 4, 5]
       },
       "expected": 10240
     },
     {
       "uuid": "5260ddde-2703-4915-b45a-e54dbbac4303",
       "property": "total",
       "description": "Check that groups of four are created properly even when there are more groups of three than groups of five",
-      "comments": ["Suggested grouping, [[1,2,3,4],[1,2,3,5],[1,2,3,4],[1,2,3,5],[1,2,3],[1,2,3]]."],
+      "comments": [
+        "Suggested grouping, [[1,2,3,4],[1,2,3,5],[1,2,3,4],[1,2,3,5],[1,2,3],[1,2,3]]."
+      ],
       "input": {
-        "basket": [1,1,1,1,1,1,2,2,2,2,2,2,3,3,3,3,3,3,4,4,5,5]
+        "basket": [
+          1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 4, 4, 5, 5
+        ]
```

```suggestion
        "basket": [1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 4, 4, 5, 5]
```

This makes copy and pasting harder for no reason.